mmv1/third_party/terraform/utils/node_config.go.erb

```diff
@@ -669,38 +688,39 @@ func flattenNodeConfig(c *container.NodeConfig) []map[string]interface{} {
 	}
 
 	config = append(config, map[string]interface{}{
-		"machine_type":             c.MachineType,
-		"disk_size_gb":             c.DiskSizeGb,
-		"disk_type":                c.DiskType,
-		"guest_accelerator":        flattenContainerGuestAccelerators(c.Accelerators),
-		"local_ssd_count":          c.LocalSsdCount,
+		"machine_type":              c.MachineType,
+		"disk_size_gb":              c.DiskSizeGb,
+		"disk_type":                 c.DiskType,
+		"guest_accelerator":         flattenContainerGuestAccelerators(c.Accelerators),
+		"local_ssd_count":           c.LocalSsdCount,
 <% unless version == 'ga' -%>
-		"ephemeral_storage_config": flattenEphemeralStorageConfig(c.EphemeralStorageConfig),
+		"ephemeral_storage_config":  flattenEphemeralStorageConfig(c.EphemeralStorageConfig),
 <% end -%>
-		"gcfs_config":              flattenGcfsConfig(c.GcfsConfig),
+		"gcfs_config":               flattenGcfsConfig(c.GcfsConfig),
 		"gvnic":										flattenGvnic(c.Gvnic),
-		"service_account":          c.ServiceAccount,
-		"metadata":                 c.Metadata,
-		"image_type":               c.ImageType,
-		"labels":                   c.Labels,
-		"tags":                     c.Tags,
-		"preemptible":              c.Preemptible,
+		"service_account":           c.ServiceAccount,
+		"metadata":                  c.Metadata,
+		"image_type":                c.ImageType,
+		"labels":                    c.Labels,
+		"tags":                      c.Tags,
+		"preemptible":               c.Preemptible,
 <% unless version == 'ga' -%>
-		"spot":                     c.Spot,
+		"spot":                      c.Spot,
 <% end -%>
-		"min_cpu_platform":         c.MinCpuPlatform,
-		"shielded_instance_config": flattenShieldedInstanceConfig(c.ShieldedInstanceConfig),
-		"taint":                    flattenTaints(c.Taints),
-		"workload_metadata_config": flattenWorkloadMetadataConfig(c.WorkloadMetadataConfig),
+		"min_cpu_platform":          c.MinCpuPlatform,
+		"shielded_instance_config":  flattenShieldedInstanceConfig(c.ShieldedInstanceConfig),
+		"taint":                     flattenTaints(c.Taints),
+		"workload_metadata_config":  flattenWorkloadMetadataConfig(c.WorkloadMetadataConfig),
 <% unless version == 'ga' -%>
-		"sandbox_config": 			flattenSandboxConfig(c.SandboxConfig),
+		"sandbox_config": 			 flattenSandboxConfig(c.SandboxConfig),
```

Comment:
nit: A couple of these use tabs instead of spaces to line up the inner block. Do you mind adjusting those, so that they're consistent with the others and use spaces? (indentation on the left side of the line should remain tabs, though!)

They were tabs in the first place before your change (and were even already mixed tabs+spaces), I figure we can clean them up pretty easily while here.